### PR TITLE
simple-web-server: new port 3.0.0-rc3

### PR DESCRIPTION
**Pull request by repagh**
#### Description

simple-web-server, a c++ (header only) library for web server development

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x ] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x ] tried a full install with `sudo port -vst install`?
- [x ] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/simple-web-server/Portfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.0
```
Comment (mf2k) [inline devel/simple-web-server/Portfile]: Please change to and test with the cmake 1.1 portgroup. 

See https://trac.macports.org/ticket/47197 for more information.

File: devel/simple-web-server/Portfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+name	  	    simple-web-server
+version		    3.0.0-rc3
+categories          devel
+license             MIT
+maintainers         gmail.com:rene.vanpaassen
```
Comment (mf2k) [inline devel/simple-web-server/Portfile]: Add your obfuscated email address. Can `openmaintainer` be added too?

File: devel/simple-web-server/Portfile [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+homepage            https://gitlab.com/eidheim/Simple-Web-Server
+master_sites	    https://gitlab.com/eidheim/Simple-Web-Server/-/archive/v${version}/
+distname	    Simple-Web-Server-v${version}
+
+checksums           rmd160 d757957fd491b0f7fe8ea6cb9cfba4a133174b2a \
+                    sha256 b6230fb1f267cf7f45b6b8eb6efb3ed6a4308fd48003dc8397eba1d7cf048467 \
+		    size    30857
+
+depends_build	    port:boost \
+		    port:cmake
```
Comment (mf2k) [inline devel/simple-web-server/Portfile]: Use `depends_build-append` here and remove `port:cmake`. The cmake portgroup adds the correct dependency but you were overriding that.

File: devel/simple-websocket-server/Portfile [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.0
```
Comment (mf2k) [inline devel/simple-websocket-server/Portfile]: The same comments on simple-web-server apply to this Portfile.

File: devel/simple-web-server/Portfile (hunk 3 above)
Comment (repagh) [reply devel/simple-web-server/Portfile]: thanks. Did that

File: devel/simple-web-server/Portfile (hunk 1 above)
Comment (repagh) [reply devel/simple-web-server/Portfile]: done

File: devel/simple-web-server/Portfile (hunk 2 above)
Comment (repagh) [reply devel/simple-web-server/Portfile]: done

File: devel/simple-websocket-server/Portfile (hunk 4 above)
Comment (repagh) [reply devel/simple-websocket-server/Portfile]: fixed 'm both

File: devel/simple-web-server/Portfile [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.1
+
+name	  	    simple-web-server
+version		    3.0.0-rc3
+categories          devel
+license             MIT
+maintainers         gmail.com:rene.vanpaassen @repagh openmaintainer
```
Comment (mf2k) [inline devel/simple-web-server/Portfile]: You have to put brackets (`{` and `}`) around your email and GitHub handle to associate them.

**Comment by mf2k** [comment]
One more minor fix and this looks about ready to go. You will need to squash your commits into 1 and just use this for the commit message:

`simple-web-server: new port
`

File: devel/simple-web-server/Portfile (hunk 5 above)
Comment (repagh) [reply devel/simple-web-server/Portfile]: done & squashed.

**Comment by kencu** [comment]
@mf2k? commit?

**Comment by mf2k** [comment]
Yes, sorry, haven't gotten back to looking at this. I think it is good to go.

**Pull request merged**